cmd/qat_plugin/qat_plugin.go

```diff
@@ -288,6 +320,7 @@ func (dp *devicePlugin) scan() (deviceplugin.DeviceTree, error) {
 func main() {
 	dpdkDriver := flag.String("dpdk-driver", "vfio-pci", "DPDK Device driver for configuring the QAT device")
 	kernelVfDrivers := flag.String("kernel-vf-drivers", "dh895xccvf,c6xxvf,c3xxxvf,d15xxvf", "Comma separated VF Device Driver of the QuickAssist Devices in the system. Devices supported: DH895xCC,C62x,C3xxx and D15xx")
+	balanceDevices := flag.Int("balance-devices", 0, "")
```

Comment:
This should be `flag.Bool()` and have a help string.

Reply:
Fixed. Help string was simply forgotten.